update idp section with new admin ui (#1515)
keycloak/keycloak-documentation: server_admin/topics/identity-broker/configuration.adoc

.Identity Providers
image:{project_images}/identity-providers.png[Identity Providers]
+
ifeval::[{project_community}==true]

> . Select card of the identity provider you want to add. {project_name} displays the configuration page for the identity provider you selected.

Reviewer: ```suggestion
. Select an identity provider. {project_name} displays the configuration page for the identity provider you selected.
```
